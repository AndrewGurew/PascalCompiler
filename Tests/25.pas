if x > 0 then modul := x else modul := -x;
if x>0 then modul:=x else modul:=-x;