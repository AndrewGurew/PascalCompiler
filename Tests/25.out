Position            Type                Text                Value               
--------------------------------------------------------------------------------
(1,1)               Key word            if                  if                  
(1,4)               ID                  x                   x                   
(1,6)               Special symbol      >                   >                   
(1,8)               Number              0                   0                   
(1,10)              ID                  then                then                
(1,15)              ID                  modul               modul               
(1,21)              Special symbol      :=                  :=                  
(1,24)              ID                  x                   x                   
(1,26)              Key word            else                else                
(1,31)              ID                  modul               modul               
(1,37)              Special symbol      :=                  :=                  
(1,40)              Special symbol      -                   -                   
(1,41)              ID                  x                   x                   
(1,42)              Special symbol      ;                   ;                   
(2,2)               Key word            if                  if                  
(2,5)               ID                  x                   x                   
(2,6)               Special symbol      >                   >                   
(2,7)               Number              0                   0                   
(2,9)               ID                  then                then                
(2,14)              ID                  modul               modul               
(2,19)              Special symbol      :=                  :=                  
(2,21)              ID                  x                   x                   
(2,23)              Key word            else                else                
(2,28)              ID                  modul               modul               
(2,33)              Special symbol      :=                  :=                  
(2,35)              Special symbol      -                   -                   
(2,36)              ID                  x                   x                   
(2,37)              Special symbol      ;                   ;                   
